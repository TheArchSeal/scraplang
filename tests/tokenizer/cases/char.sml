'a'
'"'
'\''
'\"'
'\n'
'\r'
'\t'
'\x61'
