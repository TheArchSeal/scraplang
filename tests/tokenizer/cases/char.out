(3):1:1 'a' a
(3):2:1 '"' "
(3):3:1 '\'' '
(3):4:1 '\"' "
(3):5:1 '\n' 

(3):6:1 '\r' 
(3):7:1 '\t' 	
(3):8:1 '\x61' a
